src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(String color,double firstLeg, double secondLeg) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double calculateArea() {
+        return (firstLeg * secondLeg) * 0.5;
+    }
+
+    public void draw() {
+        System.out.println("Figure: right triangle, area: " + calculateArea()
+                + " sq. units, firstLeg: " + firstLeg + " units, secondLeg: "
+                + secondLeg + " units, color: " + getColor());
+    }
```

Comment:
The `draw()` method should not be part of the `Figure` class as per the checklist. Consider creating a separate interface for drawing, as `draw()` and `calculateArea()` are conceptually different operations.